INSTALL.md

```diff
@@ -66,16 +66,25 @@ To built AMS with [PFA](https://github.com/flux-framework/PerfFlowAspect) suppor
 
 ```
 $ cd $CODE_ROOT/setup
+$ source setup_env_with_pfa.sh
 $ mkdir build; cd build
 $  cmake \
-   -DCMAKE_CXX_COMPILER=clang++ \
-   -DCMAKE_C_COMPILER=clang \
+   -DCMAKE_CXX_COMPILER=/usr/tce/packages/clang/clang-10.0.1-gcc-8.3.1/bin/clang++ \
```

Comment:
Do not use hard coded paths on the README.

Reply:
Fixed.